studies/armCartesianSpaceSquare/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,7 +22,12 @@ private void configureBindings() {
     public Command getAutonomousCommand() {
         return m_auton;
     }
-
+    public void runPeriodic() {
```

Comment:
i'll merge this as is, but you might want to move these logging statements to the subsystem periodic method instead of creating a periodic in RobotContainer.